Form-Controls/index.html

```diff
@@ -13,15 +13,50 @@ <h1>Product Pick</h1>
     </header>
     <main>
       <form>
-        <!-- write your html here-->
+        <p>
+          <label for="name">Name</label><br>
```

Comment:
You specified `for` attribute. What should you specify on `input` element in order to connect this `label` and this `input`?

Reply:
Same recommendation for the other fields.